docs/development/new-package-manager-template.md

```diff
@@ -1,106 +1,155 @@
 # New package manager questionnaire
 
-**Did you read our documentation on adding a package manager?**
+Did you read our documentation on adding a package manager?
 
 - [ ] I've read the [adding a package manager](adding-a-package-manager.md) documentation.
 
 ## Basics
 
-### Name of package manager
+### What's the name of the package manager?
 
-### What language does this support?
+### What language(s) does this package manager support?
 
 ### How popular is this package manager?
 
-### Does this language have other (competing?) package managers?
+### Are there other package managers for this language?
```

Comment:
I think people understand _competing_ better than this. Our goal is to understand if this new manager is a substitute for others

Reply:
Maybe we can ask if this manager is used as an alternative or competes with any existing supported managers which Renovate has